nautobot/docs/release-notes/version-1.1.md

```diff
@@ -64,14 +64,20 @@ Saved queries can easily be imported into the GraphiQL interface by using the ne
 
 #### Background Tasks now use Celery ([#223](https://github.com/nautobot/nautobot/issues/223))
 
-Celery has been introduced to eventually replace RQ for executing background tasks within Nautobot. All core usage of RQ has been migrated to use Celery.
+Celery has been introduced to eventually replace RQ for executing background tasks within Nautobot. All Nautobot **core** usage of RQ has been migrated to use Celery. 
+
+!!! note
+    User-defined background tasks and plugins are not part of Nautobot core functions
```

Comment:
The only way that we support creating "user-defined" background tasks is via plugins, so this reads to me as a bit redundant here and elsewhere. Perhaps:

```suggestion
    Custom background tasks implemented by plugins are not part of Nautobot core functions.
```

Reply:
Agreed.  I'll happily consolidate those.  Thanks for the clarification!

Reply:
```suggestion
    Custom background tasks implemented by plugins are not part of Nautobot core functions and therefore should be individually evaluated for their use of RQ vs Celery.
```